doc/man3/OSSL_sleep.pod

```diff
@@ -30,6 +30,9 @@ OSSL_sleep() does not return any value.
 
 OSSL_sleep() was added in OpenSSL 3.2.
 
+Before OpenSSL 3.6, the call could return early due to a signal interrupting
+the execution of the underlying system-level call.
```

Comment:
Arguably this is a bug eligible for backport, and so we wouldn't need to say this.